### ISSUE TITLE
ComposePreviewPublic not working as expected

### ISSUE THREAD

**Issue by florianPOLARSTEPS** [slackhq/compose-lints#189]
We have recently added `compose-lint` to our lint setup and started noticing that the `ComposePreviewPublic` rule is not detecting any of our `public` previews.

As to my understanding the following compose preview should trigger a warning
```kotlin
@Preview
@Composable
fun TestComposable_Preview() {
   // some composable
}
```

For my project this is not working. I am only able to create this warning by configuring the rule as such
```xml
<lint>
    <issue id="ComposePreviewPublic" severity="error">
        <option name="preview-public-only-if-params" value="false" />
    </issue>
</lint>
```

Please find a [small sample project](https://github.com/florianPOLARSTEPS/LintCheckIssue) demonstrating the problem
